frontend/src/pages/SubProjects/actions.js

```diff
@@ -101,9 +103,19 @@ export function setProjectHistoryOffset(offset) {
   };
 }
 
-export function fetchProjectHistory(projectId, offset, limit, showLoading = false) {
+export function fetchFirstProjectHistory(projectId, offset, limit, showLoading = false) {
   return {
-    type: FETCH_PROJECT_HISTORY,
+    type: FETCH_FIRST_PROJECT_HISTORY,
+    projectId,
+    offset,
```

Comment:
IMHO an action is business focused. Therefore I wouldn't expect to pass an `offset` and a `limit`. Especially not for the initial call.